paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/Form.kt

```diff
@@ -231,6 +241,62 @@ internal fun SaveForFutureUseElementUI(
     }
 }
 
+@Composable
+internal fun AfterpayClearpayElementUI(
+    enabled: Boolean,
+    element: AfterpayClearpayHeaderElement
+) {
+    val context = LocalContext.current
+
+    FlowRow(
+        modifier = Modifier.padding(4.dp, 8.dp, 4.dp, 4.dp),
+        crossAxisAlignment = FlowCrossAxisAlignment.Center
+    ) {
+        Text(
+            element.getLabel(context.resources),
+            Modifier
+                .padding(end = 4.dp),
+            color = if (isSystemInDarkTheme()) {
```

Comment:
Let's keep styling logic in a separate data class.  For example for the TextField I have the `TextFieldColors` data class